test/quantization/core/test_backend_config.py

```diff
@@ -37,26 +38,83 @@ class TestBackendConfig(QuantizationTestCase):
         is_dynamic=True
     )
 
-    dtype_config_dict1 = {
+    activation_dtype_with_constraints = DTypeWithConstraints(
+        dtype=torch.quint8,
+        quant_min=0,
```

Comment:
should this be named something like `quant_min_min`?  The name of the attribute is `quant_min`, is this testing equality or "less than"?

Reply:
Technically that's true, but I find that naming confusing, since it would imply there's a `quant_min_max` or a `quant_max_min`. I prefer to just hide this detail from the user and just document it clearly instead

Reply:
Right, however below we have `scale_min`.  We should be consistent, the scheme we are using is `{var_name}_{max|min}`.  I'd recommend changing this so all the restrictions are named consistently.  `quant_min_max` and `quant_max_min` are normal in this naming scheme.

Reply:
Ok, spoke with @jerryzh168 offline about this and we decided `quant_min_lower_bound` is even clearer. I'll make this change for all the fields